### PR TITLE
Web Inspector: Audit tab change the font-size of scope bars from px to rem

### PR DESCRIPTION
**Pull request by francescorn**
#### 6cc30e6c79910867e23c0c328e5d80cc677a1ba5
<pre>
Web Inspector: Audit tab change the font-size of scope bars from px to rem
<a href="https://bugs.webkit.org/show_bug.cgi?id=268345">https://bugs.webkit.org/show_bug.cgi?id=268345</a>

Reviewed by NOBODY (OOPS!).

Replace px with the equivalent rem value in `font-size` in the scope bar.

Amend changes:
* Source/WebInspectorUI/UserInterface/Views/ScopeBar.css:
(.scope-bar &gt; li):

Combined changes:
* Source/WebInspectorUI/UserInterface/Views/ScopeBar.css:
(.scope-bar &gt; li):
</pre>
https://github.com/WebKit/WebKit/commit/6cc30e6c79910867e23c0c328e5d80cc677a1ba5

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/37061 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/48/builds/15973 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/14/builds/39366 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/39643 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/32/builds/33114 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/49/builds/18614 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/51/builds/13106 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/31626 "Passed tests") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/37623 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/47/builds/13535 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/32667 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/11739 "Passed tests") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/42/builds/11793 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/10/builds/33147 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/40899 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/33633 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/36/builds/33525 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/37650 "Passed tests") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/44/builds/12063 "Built successfully") | [❌ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/50/builds/9864 "Found 1 new test failure: ipc/message-listener-async-message-reply-id.html (failure)") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/35784 "Passed tests") | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/46/builds/13706 "Built successfully") | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/43/builds/12436 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/45/builds/12970 "Built successfully") | | |

### CODE REVIEW

File: LayoutTests/imported/w3c/web-platform-tests/svg/linking/scripted/a.text-setter-01-expected.txt [hunk 1]
```diff
@@ -1,7 +1,7 @@
 
-PASS SVGAElement.text setting
-FAIL Test for anchor 0 assert_equals: expected "x" but got "a b c"
-FAIL Test for anchor 1 assert_equals: expected "x" but got "a  c"
-FAIL Test for anchor 2 assert_equals: expected "x" but got "a b c"
-FAIL Test for anchor 3 assert_equals: expected "x" but got "a b c "
```
Comment (nt1m) [inline LayoutTests/imported/w3c/web-platform-tests/svg/linking/scripted/a.text-setter-01-expected.txt]: unrelated?

**Comment by francescorn** [comment]
@n1m, I messed up in the terminal, I am reverting the changes in this pr. It should be in the other pr I am working on.

**Comment by francescorn** [comment]
Attached is an image of the before and after with this change. The mini-browser is on the left and Safari 17.3 is on the right. 
scope_bar_font-size_from_px_to_rem

**Comment by francescorn** [comment]
Ready for a review.

File: Source/WebInspectorUI/UserInterface/Views/ScopeBar.css [hunk 2]
```diff
@@ -40,7 +40,7 @@
     display: inline-block;
     margin: 0 var(--scope-bar-margin);
     padding: 2px var(--scope-bar-padding) 4px;
-    font-size: 11px;
+    font-size: 0.7rem;
```
Comment (dcrousso) [inline Source/WebInspectorUI/UserInterface/Views/ScopeBar.css:43]: this will likely have much more far reaching consequences than just in the Audit Tab since `WI.ScopeBar` is used all over the place

also, what is the benefit of switching from `px` to `rem`?

**Review by dcrousso — CHANGES_REQUESTED** [review]

File: Source/WebInspectorUI/UserInterface/Views/ScopeBar.css (hunk 2 above)
Comment (francescorn) [reply Source/WebInspectorUI/UserInterface/Views/ScopeBar.css:43]: I am going to draft this until command +/- is implemented in Web Inspector.

Comment (francescorn) [reply Source/WebInspectorUI/UserInterface/Views/ScopeBar.css:43]: It will not make too much of a difference until Command Shift  +/- is implemented in the Web Inspector.
